### PR TITLE
Added new python exercise with solution

### PR DESCRIPTION
**Pull request by Ayobami6**
Adding more python exercise and solution

### CODE REVIEW

File: topics/python/solutions/sort_solution.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+## Algorithms
```
Comment (bregman-arie) [inline topics/python/solutions/sort_solution.md:1]: Algorithms is quite generic header, maybe something with "sort"?

Comment (Ayobami6) [reply topics/python/solutions/sort_solution.md:1]: Ok

**Review by bregman-arie — APPROVED** [review]

**Comment by bregman-arie** [comment]
Thank you.

**Comment by Ayobami6** [comment]
I'm glad, Thank you

**Pull request merged**